src/AcceptanceTests/HostApplicationBuilder/When_logging_with_host_application_builder.cs

```diff
@@ -43,8 +43,8 @@ public async Task Should_integrate_with_default_host_logging()
                 await host.StartAsync();
 
                 var actual = builder.ToString();
-                StringAssert.Contains(ExpectedLogMessage, actual);
-                StringAssert.DoesNotContain(NotExpectedLogMessage, actual);
+                Assert.That(actual, Does.Contain(ExpectedLogMessage));
```

Comment:
Interesting that here Assert.Multiple is not applied. I still don't get that but 🤷

Reply:
that is indeed odd